integration/manualtests/manual_tests.go

```diff
@@ -0,0 +1,84 @@
+package manualtests
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/go-obscuro/go/ethadapter"
+	"github.com/obscuronet/go-obscuro/go/obsclient"
+	"github.com/obscuronet/go-obscuro/go/rpc"
+	"github.com/obscuronet/go-obscuro/go/wallet"
+
+	gethlog "github.com/ethereum/go-ethereum/log"
+)
+
+func awaitL1Tx(ethClient ethadapter.EthClient, signedTx *types.Transaction) error {
```

Comment:
I feel this is and awaitL2Tx can be combined since both clients should satisfy the basic client interface

Reply:
Prolly can, is the do you know if :
`Currently when a receipt is not available the obscuro node is returning nil instead of err ethereum.NotFound`
is fixed atm ?